modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/viewmodel/PlayerViewModel.kt

```diff
@@ -537,7 +537,20 @@ class PlayerViewModel @Inject constructor(
     }
 
     private fun calcCustomTimeText(): String {
-        return context.resources.getString(LR.string.minutes_plural, sleepCustomTimeMins)
+        val hours = sleepCustomTimeInMinutes / 60
+        val minutes = sleepCustomTimeInMinutes % 60
+
+        return if (hours == 1 && minutes == 0) {
+            context.resources.getString(LR.string.hours_singular)
+        } else if (hours > 1 && minutes == 0) {
+            context.resources.getString(LR.string.hours_plural, hours)
+        } else if (hours > 0) {
+            context.resources.getString(LR.string.hours_and_minutes, hours, minutes)
```

Comment:
Should this support the hour being a singular and be "1 hour and 50 minutes"?

<img width="389" alt="Screenshot 2024-05-06 at 2 07 47 PM" src="https://github.com/Automattic/pocket-casts-android/assets/308331/99466e36-c370-43ff-a81b-02aefc1222ac">

Reply:
Yes! I've jus added here: [a07648b](https://github.com/Automattic/pocket-casts-android/pull/2160/commits/a07648bba290925853350fc3e3efd75edd42febc)